qa-core/package.json

```diff
@@ -17,7 +17,8 @@
     "test:smoke": "yarn run test --testPathIgnorePatterns=/.+\\.integration\\.spec\\.ts",
     "test:ci": "start-server-and-test dev:built http://localhost:4001/health test:smoke",
     "serve": "start-server-and-test dev:built http://localhost:4001/health",
-    "dev:built": "cd ../ && yarn dev:built"
+    "dev:built": "cd ../ && yarn dev:built",
+    "test:datasources": "jest --runInBand --json --outputFile=testResults.json --testPathPattern='datasources/.*\\.spec\\.(ts|js)$'"
```

Comment:
We can probably remove the `test:smoke` now? I take it this is the replacement. How does it affect the commands for the test that run against QA env?